### ISSUE TITLE
VCF output for xtea_long?

### ISSUE THREAD

**Issue by eyalmpeer** [parklab/xTea#29]
Hello and thank you for sharing this tool.
I ran xtea_long, per the instructions on the xtea_long branch, on Pac Bio data.
The final output was only txt files.
Is it possible to generate the VCF files mentioned in the article that can aid in determining the zygosity of the insertions?
Or any way to extract from the xtea_long output how many reads in the insertion location support the insertion and how many reads do not support it?
Thanks.

**Comment by simoncchu** [comment]
Yeah, this is in my to-do-list. I'll export a vcf file format. For the current output, each column representation could be find here: https://github.com/parklab/xTea_paper/tree/main/run_tools/xTea/HG002. 
There is a intermediate file called `candidate_list_from_clip.txt` has the number of clipped reads (third column), but I didn't count the mapped...

**Comment by xzhuo** [comment]
Thanks for your swift reply! Are there ways to infer the correct consensus position for SVA?

**Comment by simoncchu** [comment]
It's not straightforward as the reference SVA annotation is fragmented and inaccurate (because of the tandem repeats expansion). For a simple way, just consider position 0 as the start position on the consensus, but it may be inaccurate.
